examples/text_to_image/train_text_to_image.py

```diff
@@ -691,6 +710,40 @@ def collate_fn(examples):
                         accelerator.save_state(save_path)
                         logger.info(f"Saved state to {save_path}")
 
+                        if args.validation_prompt:
+                            pipeline = StableDiffusionPipeline.from_pretrained(
+                                args.pretrained_model_name_or_path,
+                                text_encoder=text_encoder,
+                                vae=vae,
+                                unet=unet,
+                                revision=args.revision,
+                            )
+                            pipeline = pipeline.to(accelerator.device)
+                            pipeline.set_progress_bar_config(disable=True)
+
+                            # run inference
+                            prompt = [args.validation_prompt]
+                            with torch.autocast("cuda"):
```

Comment:
```suggestion
                            with torch.autocast(accelerator.device, enabled=args.mixed_precision == "fp16"):
```

Courtesy: @patil-suraj (https://github.com/huggingface/diffusers/pull/2173#pullrequestreview-1277057469)